### PR TITLE
[14.0][MIG] product_supplierinfo_for_customer_picking

### PR DESCRIPTION
**Pull request by tafaRU**
Locked by https://github.com/OCA/product-attribute/pull/859

### CODE REVIEW

File: product_supplierinfo_for_customer_picking/models/stock_move.py [hunk 1]
```diff
@@ -1,30 +1,33 @@
...
+                move.product_customer_code = False
+                continue
+            else:
+                customers = move.product_tmpl_id.customer_ids.filtered(
+                    lambda m: move.picking_id.partner_id
+                )
+                product_customer_code = (
+                    customers[0].product_code if customers else False
+                )
+                move.product_customer_code = product_customer_code
```
Comment (fredzamoabg) [inline product_supplierinfo_for_customer_picking/models/stock_move.py]: you can probably perform refactoring here with something like:

          ```
          product_customer_code = False
            if (
                move.picking_id
                and move.picking_id.partner_id
                and move.product_tmpl_id.customer_ids
            ):
                customer = fields.first(move.product_tmpl_id.customer_ids.filtered(
                    lambda m: move.picking_id.partner_id
                ))
                product_customer_code = customer.product_code
            move.product_customer_code = product_customer_code
```

what do you think?

**Review by fredzamoabg — CHANGES_REQUESTED** [review]

File: product_supplierinfo_for_customer_picking/models/stock_move.py (hunk 1 above)
Comment (tafaRU) [reply product_supplierinfo_for_customer_picking/models/stock_move.py]: Definitely better! Done, thanks.

**Review by fredzamoabg — APPROVED** [review]

**Comment by SimoRubi** [comment]
@OCA/logistics-maintainers can this be merged? Thanks!

**Comment by tafaRU** [comment]
@rousseldenis are you ok with merging this?

**Review by rousseldenis — APPROVED** [review]

**Pull request merged**